runtime/compiler/x/codegen/AllocPrefetchSnippet.cpp

```diff
@@ -87,7 +87,7 @@ uint8_t *TR::X86AllocPrefetchSnippet::emitSnippetBody()
       {
       TR_RuntimeHelper helper = (comp->getOption(TR_EnableNewX86PrefetchTLH)) ? TR_X86newPrefetchTLH : TR_X86prefetchTLH;
       helperSymRef = cg()->symRefTab()->findOrCreateRuntimeHelper(helper);
-      disp32 = cg()->branchDisplacementToHelperOrTrampoline(buffer+4, helperSymRef);
+      disp32 = cg()->branchDisplacementToHelperOrTrampoline(buffer-1, helperSymRef);
```

Comment:
Instead of doing `buffer-1` here, couldn't the code above be changed from
```
*buffer++ = 0xe8;
```
to
```
*buffer = 0xe8;
```
That way all other changes follow the theme of this PR, namely changing the `+4` to `+5`, and doing a `++buffer` when writing the immediate.

Reply:
Doing it this way I only had to update one spot in the code, but I can see there's an argument to be made for consistency.  I can change it.

Reply:
Spoke offline, since this is a code clean up suggestion, I suggested we leave it for another PR to reduce having to run a whole suite of tests again.

Reply:
I discussed with @dsouzai and this will be cleaned up in a subsequent PR.